source/includes/_personnel.md

```diff
@@ -0,0 +1,411 @@
+
+### Request
+
+`GET https://api.handshq.com/v1/personnel`
+
+### Allowed Query Parameters
+Parameter | Format | Required | Description
+--------- | ------ | -------- | -----------
+search    | String | No       | Provide a search field to search the first name, last name or email address of personnel
+archived  | String | No       | Provide a param of true to fetch only archived personnel
+
+### Response
+
+Successful requests will return a collection of personnel and a `200` status code.
+Results in `data` are [paginated](#pagination)
+
+## Viewing one personnel
+
+```shell
+curl https://api.handshq.com/v1/personnel/[id] \
+  -H "Accept: application/json" \
+  -H "Authorization: bearer [api_token]"
+```
+
+> 200
+
+```json
+  {
+    "data": {
+      "id": "1234",
+      "type": "personnel",
+      "attributes": {
+        "first_name": "John",
+        "last_name": "Smith",
+        "email": "john.smith@email.com",
+        "archived_at": null,
+        "type": "employee",
+        "training_status": {
+          "status": "missing",
+          "description": "missing training"
+      },
+      "relationships": {
+        "line_manager": {
+          "data": {
+            "id": "4321",
+            "type": "line_manager"
+          }
+        },
+        "roles": {
+          "data": [
+            {
+              "id": "123",
+              "type": "role"
+            },
+            {
+              "id": "321",
+              "type": "role"
+            }
+          ]
+        }
+      }
+    }
+  }
+```
+
+This endpoint allows you to view a personnel by providing the id.
+
+### Request
+
+`GET https://api.handshq.com/v1/personnel/[id]`
+
+### Response
+
+Successful requests will return a json payload of that personnel and a `200` status code
+
+## Creating a personnel
+
+```shell
+curl https://api.handshq.com/v1/personnel \
+  -H "Accept: application/json" \
+  -H "Authorization: bearer [api_token]" \
+  -H "Content-Type: application/json" \
+  --request POST \
+  -d '[json_payload]'
+```
+
+> Example Personnel creation payload.
+
+```json
+  {
+    "personnel": {
+      "first_name": "Sandra",
+      "last_name": "Smith",
+      "email":"sandra.smith@email.com",
+      "line_manager_id": "456",
+      "role_ids": ["987", "765"]
+    }
+  }
+```
+
+This endpoint allows you to create a personnel. Personnel are division specific, so the personnel will belong to the division that the API key belongs to. You can set the first name, last name and email of the personnel through this endpoint. You can also assign the personnel a line manager using the ID of another personnel, and assign roles to the personnel using the IDs of roles that exist in the account.
+
```

Comment:
By convention I think if we have to anything that the user can configure then we should be describing it in tabular form (like the Allowed Project Parameters part) where we can describe each attribute if needed

Reply:
Cool, I have added this to personnel, role and personnel_role